api-definitions/db_positions/api.yaml

```
@@ -0,0 +1,292 @@
+  contact:
+    name: PagoPA
+    url: https://www.pagopa.gov.it/
+  x-logo:
+    url: https://www.pagopa.gov.it/assets/images/pagopa-logo.png
+  description: Specification for Debtor Positions Interfaces
+host: api.io.italia.it.TOBEDEFINE
+basePath: "/api/v1"
+schemes:
+  - https
+security:
+  - ApiKey: []
+paths:
+  "/payment-notices/{organizationfiscalcode}-{paymentnoticenumber}":
+    post:
+      operationId: createDbPosition
+      summary: |
+        The PA creates a Debt Position
+      parameters:
+        - $ref: "#/parameters/OrganizationFiscalCode"
+        - $ref: "#/parameters/PaymentNoticeNumber"
+        - name: payment-posistion
+          in: body
+          required: true
+          schema:
+            $ref: "#/definitions/PaymentPosition"
+      responses:
+        "201":
+          description: Request created.
+          schema:
+            $ref: "#/definitions/PaymentPosition"
+        "401":
+          description: Wrong or missing function key.
+        "409":
+          description: "Conflict: pending request found."
+          schema:
+            $ref: "#/definitions/ProblemJson"
+        "500":
+          description: Service unavailable.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+    get:
+      operationId: getDbPosition
+      summary: |
+        The PA gets the details of a specific debt position, or searches for all its PDs that meet a certain search criterion.
+      parameters:
+        - $ref: "#/parameters/OrganizationFiscalCode"
+        - $ref: "#/parameters/PaymentNoticeNumber"        
+      responses:
+        "200":
+          description: payment position details.
+          schema:
+            $ref: "#/definitions/PaymentPosition"
+        "401":
+          description: Wrong or missing function key.
+        "404":
+          description: No payment found.
+        "500":
+          description: Service unavailable.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+    delete:
+      operationId: delDbPosition
+      summary: |
+        ThePA can only cancel its own PDs, or rather the entities it manages.
+      parameters:
+        - $ref: "#/parameters/OrganizationFiscalCode"
+        - $ref: "#/parameters/PaymentNoticeNumber"        
+      responses:
+        "200":
+          description: payment position details.
+          schema:
+            $ref: "#/definitions/PaymentPosition"
+        "401":
+          description: Wrong or missing function key.
+        "404":
+          description: No payment found.
+        "500":
+          description: Service unavailable.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+    patch:
+      operationId: updateDbPosition
+      summary: |
+        The PA update a Debt Position
+      parameters:
+        - $ref: "#/parameters/OrganizationFiscalCode"
+        - $ref: "#/parameters/PaymentNoticeNumber"
+      responses:
+        "204":
+          description: Request updated.
```

```suggestion
          description: Payment position updated.
```

I' try to use always the same names (payment position VS debt position vs PD, ...)